### ISSUE TITLE
Cannot change the default route path

### ISSUE THREAD

**Issue by GautierT** [ForestAdmin/forest-express-mongoose#36]
Hello,
I would like to move the default route path to /api/forest.
In my forest admin interface i change the default endpoint to end with /api 
then in my app.js i initialize forest like this : 
```
app.use('/api', require('forest-express-mongoose').init({
    modelsDir: __dirname + '/models',  // Your models directory.
    envSecret: config.forest.env_secret,
    authSecret: config.jwt.secret,
    mongoose: mongoose // The database connection.
}));
```
But then the password doesn't work and i have that in my log : `[forest] 🌳🌳🌳  Bad JWT token.`
And indeed when i console.log req.user i can see that it's undefined.

**Comment by arnaudbesnier** [comment]
hey @GautierT thank you for the issue.
You are right, the "custom path" is not properly handled by the package right now.

I have a fix for that, I will make a pull request today.

**Comment by arnaudbesnier** [comment]
Fixed in the PR and will be released soon.

**Comment by arnaudbesnier** [comment]
The fix is now released in the liana version 0.3.2.
